src/main/java/model/students/Student.java

```diff
@@ -1,11 +1,19 @@
 package model.students;
 
+import model.exceptions.NotValidEmailException;
+import model.exceptions.NotValidPhoneException;
+
 import java.util.Comparator;
 
+import static util.Validator.isEmailValid;
```

Comment:
to improve readability it's better to to use this methods as Validator.isEmailValid(..) instead of importing the method. It looks more clear